src/components/pages/cross_word/CrossWordGame/crossword-game.module.css

```
@@ -0,0 +1,105 @@
+    color-mix(in oklch, var(--muted-foreground) 18%, transparent) 5px,
+    color-mix(in oklch, var(--muted-foreground) 18%, transparent) 6px
+  );
+}
+
+/* ── Word-accepted pulse — fires once per solve ── */
+/* Cannot be expressed in Tailwind: multi-step box-shadow + scale keyframe */
+.cellJustSolved {
+  animation: solvedPulse 0.55s cubic-bezier(0.25, 0.46, 0.45, 0.94);
+}
+
+@keyframes solvedPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "solvedPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)